marklogic-data-hub-central/ui/e2e/cypress/integration/curation/curate/mappingWithInterceptorsAndCustomHook.spec.tsx

```diff
@@ -115,7 +115,7 @@ describe("Create and verify load steps, map step and flows with interceptors & c
     advancedSettingsDialog.setStepInterceptor("curateTile/orderDateInterceptor");
     createEditMappingDialog.saveButton().click({force: true});
     cy.waitForAsyncRequest();
-    cy.waitUntil(() => curatePage.dataPresent().should("be.visible"));
+    cy.waitUntil(() => curatePage.dataPresent().should("exist"));
```

Comment:
Do we need this "waitUntil" for some reason? The **should** function already performs a wait itself until the element (in this case) _exists_. If you need more time until this element _exists_, you can set a bigger timeout for the element (let me know if you need help with this :) )

Reply:
Just out of curiosity, the "be.visible" was not working anymore for some reason? Or it did not have enough time until it was visible and it was failing first? If that's the issue maybe it can be fixed with the suggestion I shared above (because if we only wait until it exists, but not until it's visible maybe some things can break later because another action needs this element visible. I did not analyze the whole test, this is just a thought, I hope it's helpful! If not then just simply ignore me :) )

Reply:
Discussed with Camila that this will be investigated in a separate task, resolving.